free-2.md

```diff
@@ -1,44 +1,51 @@
-# Free project 2
-
+# Legislative Law
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The language is designed for policy makers and legislators. The legislative process for passing laws seems to be concerned less with the content of the law and more with the language of the law. Several laws are written with intentional vagueness and ambiguity for the sake of less opposition. This turns out to be very problematic later on as ambiguity inevitably leads to different interpretations with respect to legality. This language would seek to provide some standard for future laws and if permitted some form of logic checking.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea helps eliminate ambiguity in laws. This language would help legislators in both developing laws and referencing laws. With a computer based language, it would be significantly easier to look up laws. For instance, there may be several laws that come up when specific key words are searched(e.g. "shoplifting with weapon law") that differ drastically based on context (state-state level, scale of the shope, state-national level). Additionally, legislators could avoid drafting a similar law that was previously repealled or still enact. As of now, it seems like drafting a law requires a significant amount of man power just to reference check thoroughly. 
 
```

Comment:
This seems like an interesting idea, though I think that it might get pretty confusing or ambiguous to delve into NLP and it could quickly get out of hand.